docs/overview.md

```diff
@@ -6,12 +6,17 @@
 
 # TensorFlow I/O
 
-TensorFlow I/O is a collection of file systems and file formats that are not
-available in TensorFlow's built-in support.
-
-TensorFlow I/O has integrations with many systems and cloud vendors such as
-Prometheus, Apache Kafka, Apache Ignite, Google Cloud PubSub, AWS Kinesis,
-Microsoft Azure Storage, Alibaba Cloud OSS etc.
+TensorFlow I/O is an extension package to Tensorflow, which encompasses io support for 
+a collection of file systems and file formats that are not available in TensorFlow's built-in support.
+Integrations with many systems and cloud vendors include (but not limited to):
+
+- Prometheus
+- Apache Kafka
+- Apache Ignite
+- Google Cloud PubSub
```

Comment:
Since you already updating this, would you mind adding Google Cloud BigQuery here?
It is used more widely than PubSub.

Reply:
Added in the new commit.